systemtest/src/test/java/io/strimzi/systemtest/kafka/TieredStorageST.java

```diff
@@ -115,7 +123,7 @@ void testTieredStorageWithAivenPlugin() {
             .endSpec()
             .build());
 
-        resourceManager.createResourceWithWait(KafkaTopicTemplates.topic(testStorage.getClusterName(), testStorage.getTopicName(), suiteStorage.getNamespaceName())
+        KafkaTopicBuilder kt = KafkaTopicTemplates.topic(testStorage.getClusterName(), testStorage.getTopicName(), suiteStorage.getNamespaceName())
```

Comment:
why it is passed to the builder and then just used in the `createResourceWithWait`? You can keep it as it was or?

Reply:
Leftover copy/paste, removed